src/librustdoc/html/markdown.rs

```diff
@@ -1413,7 +1429,11 @@ impl MarkdownWithToc<'_> {
             html::push_html(&mut s, p);
         }
 
-        format!("<nav id=\"TOC\">{toc}</nav>{s}", toc = toc.into_toc().print())
+        (toc.into_toc(), s)
+    }
+    pub(crate) fn into_string(self) -> String {
+        let (toc, s) = self.into_parts();
+        format!("<nav id=\"TOC\">{toc}</nav>{s}", toc = toc.print())
```

Comment:
Just one thought: this ID and `ModNav` are the only IDs with capital letters. A reason for that?

Reply:
Yes please. OCD triggering hard on this one. ^^'

Reply:
Oh I see. Can you update `ModNav` at least please?

Reply:
👍